config.py

```diff
@@ -12,17 +12,18 @@
 embedding_range_dict = {
     "2": 2,
     "10": 3,
-    "1000": 6,
```

Comment:
is this bringing any change? I guess the original values were conceptually more aligned with what we want to achieve?